src/webhook_launcher/app/models.py

```diff
@@ -244,6 +244,11 @@ class WebHookMapping(models.Model):
         help_text="Choose Y to take content of revision as-is without "
                   "automatic processing (example: tarballs in git)",
     )
+    placeholder = models.BooleanField(
+        default=False,
+        editable=False,
+        help_text="Marks automatically created placehollders",
```

Comment:
As of now it is only intended as marker for filtering in the admin UI, and it is automatically set to false when the WHM is updated.

For triggers it doesn't make difference if the WHM is a placeholder or not. Placeholders are created with build==False and when you change that, the WHM is no longer marked as placeholder.